rust-bindings/src/auto/functions.rs

```diff
@@ -82,7 +82,7 @@ pub fn commit_get_object_sizes(commit_variant: &glib::Variant) -> Result<Vec<Com
 }
 
 #[doc(alias = "ostree_commit_get_parent")]
-pub fn commit_get_parent(commit_variant: &glib::Variant) -> glib::GString {
+pub fn commit_get_parent(commit_variant: &glib::Variant) -> Option<glib::GString> {
```

Comment:
Verified we want this API change, and the other ones here too.

I think I accidentally dropped a commit to regenerate the Rust bindings in my previous PR.